app/controllers/album.js

```diff
@@ -1,7 +1,62 @@
-const albumService = require('../services/album');
+const albumService = require('../services/album'),
+  albumInteractor = require('../interactors/album'),
+  error = require('../errors'),
+  logger = require('../logger'),
+  config = require('../../config'),
+  { validationResult } = require('express-validator/check');
 
 exports.list = (req, res, next) => {
 
-  albumService.listRequest(req, res, next);
+  logger.info(`Attempting GET request to url ${config.common.urlRequests.albumList}`);
+
+  albumService.listRequest().then(albums => {
+    logger.info(`Response received from ${config.common.urlRequests.albumList}`);
```

Comment:
Same here I would leave a comment more related to what the controllers knows, to print a comment with the url that the service is using here seems like a strange. Anyways it is still a comment, but I think it helps getting to understand a little bit more what each part of the code is responsable of.